### PR TITLE
Add `NewRelease.GenerateReleaseNotes` property

### PR DESCRIPTION
**Pull request by mryanmurphy**
Enable generating release notes during creation of a release.

### CODE REVIEW

File: Octokit/Models/Request/NewRelease.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ namespace Octokit
     /// Used to create a new release.
     /// </summary>
     /// <remarks>
-    /// API: https://developer.github.com/v3/repos/releases/#create-a-release
+    /// API: https://docs.github.com/en/rest/releases/releases#create-a-release
```
Comment (JonruAlveus) [inline Octokit/Models/Request/NewRelease.cs:10]: Please could you remove the en from the url so GitHub can work out the language appropriately?

File: Octokit/Models/Request/NewRelease.cs [hunk 2]
```diff
@@ -32,7 +32,7 @@ public NewRelease(string tagName)
 
         /// <summary>
         /// Specifies the commitish value that determines where the Git tag is created from. Can be any branch or
-        /// commit SHA. Unused if the Git tag already exists. Default: the repository�s default branch
+        /// commit SHA. Unused if the Git tag already exists. Default: the repository’s default branch
```
Comment (JonruAlveus) [inline Octokit/Models/Request/NewRelease.cs:35]: I think this is an encoding issue? Shouldn’t it be an apostrophe character?

File: Octokit/Models/Request/ReleaseUpdate.cs [hunk 3]
```diff
@@ -7,7 +7,7 @@ namespace Octokit
     /// Used to update a release.
     /// </summary>
     /// <remarks>
-    /// API: https://developer.github.com/v3/repos/releases/#create-a-release
+    /// API: https://docs.github.com/en/rest/releases/releases#update-a-release
```
Comment (JonruAlveus) [inline Octokit/Models/Request/ReleaseUpdate.cs:10]: Please could you remove the en from the url so GitHub can work out the language appropriately?

**Review by JonruAlveus — CHANGES_REQUESTED** [review]

File: Octokit/Models/Request/NewRelease.cs (hunk 2 above)
Comment (mryanmurphy) [reply Octokit/Models/Request/NewRelease.cs]: Reverted, this was an unintentional change.

**Comment by JonruAlveus** [comment]
LGTM! ❤️ Thanks for the contribution!

**Review by JonruAlveus — APPROVED** [review]

**Comment by JonruAlveus** [comment]
release_notes: Add NewRelease.GenerateReleaseNotes property

**Pull request merged**